.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,3 +1,5 @@
+**TLDR;** Please include a short description (1-2 sentences) about the changes you are introducing. This content will be used in our release notes for Dawn on [themes.shopify.com](https://themes.shopify.com/themes/dawn/styles/default#ReleaseNotes).
```

Comment:
For things to be consistent with what the current PR template, one suggestion would be to change this part to the following:

**What changes are you introducing?** _Please include a short description (1-2 sentences). This content will be used in our release notes for Dawn on [themes.shopify.com](https://themes.shopify.com/themes/dawn/styles/default#ReleaseNotes)_